ansible_collections/arista/avd/docs/installation/collection-installation.md

```diff
@@ -1,14 +1,19 @@
-# Collection Installation
+# Collection installation
 
-These instructions are for regular users to install via Ansible Galaxy. To setup a development environment use [these](../contribution/setup-environment.md) instructions.
+## Installation workflow
```

Comment:
With this change it might make more sense to gather everything in this installation guide instead of pointing some steps to the requirements page.

Reply:
Yes, we had considered that, but didn't want to make to many changes.
I agree and we should do this now!